packages/contracts/contracts/CdpManagerStorage.sol

```diff
@@ -456,33 +451,6 @@ contract CdpManagerStorage is EbtcBase, ReentrancyGuard, ICdpManagerData, AuthNo
         return stake;
     }
 
-    /*
-     * Remove a Cdp owner from the CdpOwners array, not preserving array order. Removing owner 'B' does the following:
-     * [A B C D E] => [A E C D], and updates E's Cdp struct to point to its new array index.
-     */
-    function _removeCdp(bytes32 _cdpId, uint256 cdpIdsArrayLength) internal {
-        Status cdpStatus = Cdps[_cdpId].status;
-        // It’s set in caller function `_closeCdp`
-        require(
-            cdpStatus != Status.nonExistent && cdpStatus != Status.active,
-            "CdpManagerStorage: remove non-exist or non-active CDP!"
-        );
-
-        uint128 index = Cdps[_cdpId].arrayIndex;
-        uint256 length = cdpIdsArrayLength;
-        uint256 idxLast = length - 1;
-
-        require(index <= idxLast, "CdpManagerStorage: CDP indexing overflow!");
-
-        bytes32 idToMove = CdpIds[idxLast];
-
-        CdpIds[index] = idToMove;
-        Cdps[idToMove].arrayIndex = index;
-        emit CdpArrayIndexUpdated(idToMove, index);
```

Comment:
The `function _removeCdp()`, which contained `emit CdpArrayIndexUpdated(idToMove, index);`, has been removed. However, it seems that the corresponding event type definition for the now-unused `CdpArrayIndexUpdated` may have been inadvertently left undeleted.